### PR TITLE
F/[B2B-171][B2D-328][B2D-28][B2B-23]

### PR DESCRIPTION
**Pull request by sandrom**
please do not merge instantly, this has to be synched with changes that have to be made by @YuraTolochkevich

### CODE REVIEW

File: pipeline/my-first-actor/build.sbt [hunk 1]
```diff
@@ -9,14 +7,24 @@ scalaVersion := "2.11.8"
...
+  "io.geeny" %% "commons" % "0.1-SNAPSHOT",
 
+  "org.scalatest" %% "scalatest" % "3.0.0" % Test
 )
 
-assemblyMergeStrategy in assembly := {
-  case PathList("META-INF", xs @ _*) => MergeStrategy.discard
-  case n if n.startsWith("reference.conf") => MergeStrategy.concat
-  case _ => MergeStrategy.first
+assemblyExcludedJars in assembly := {
```
Comment (dmantula) [inline pipeline/my-first-actor/build.sbt]: Why don't you just use "io.geeny" %% "commons" % "0.1-SNAPSHOT" % Provided, ?

Comment (sandrom) [inline pipeline/my-first-actor/build.sbt]: because provided bugs in idea and this is a really easy way to get rid of it without idea bugging

Comment (sandrom) [inline pipeline/my-first-actor/build.sbt]: also it is not necessarily provided by a runtime, just in assembly it should be gone to work nicely with our current way of assembling. this whole thing will go away once we dont do it that stupid anymore someday, but we still do for now :)

Comment (dmantula) [inline pipeline/my-first-actor/build.sbt]: As for bug in Idea – it's not an argument.
As for 'necessarily provided by a runtime' – it is necessarily.
As for 'once we dont do it that stupid anymore someday' – what exactly do you mean? Are you going to develop geeny-sbt plugin? Or how we are going to change developer-facing part?

**Comment by edimoldovan** [comment]
What’s up with this PR? Is it going in, changes will be done, or not? cc @platy @felixroeser

**Comment by sandrom** [comment]
@edimoldovan it was waiting for dmytro to approve, which he didnt wanna do without a cosmetic change. i did it anyway even though there are downsides to it, which hopefully wont apply here, so we can hopefully finally merge this now

**Pull request merged**